parkings/models/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .addresses import *
+from .operators import *
+from .parkings import *
```

Comment:
Sure, can do 👍 It's quite common imho to use the combo of `*` import in `__init__.py` and then define `__all__` in the files themselves (to not create that pollution), but explicit without __all__ is probably clearer as said.